chris_truax/mean_stack/js_OOP/ninja2.js

```diff
@@ -0,0 +1,39 @@
+function Ninja(name, health=100){
+  this.name = name;
+  this.health = health;
+  var speed = 3;
+  var strength = 3;
+  this.sayName = function(){
+    console.log('My ninja name is: ' + this.name);
+    return this;
+  }
+  this.showStats = function(){
+    console.log('Name: '+ this.name + ', Speed: ' + speed + ', Health: ' + this.health + ', Strength: ' + strength);
+    return this;
+  }
+  this.drinkSake = function(){
+    this.health += 10;
+    return this;
+  }
+  this.punch = function(opponent){
+    if(opponent instanceof Ninja == true){
+      opponent.health -= 5;
+      console.log(opponent.name + ' was punched by ' + this.name + ' and lost 5 teeth!');
+    }
+  }
+  this.kick = function(opponent){
+    if(opponent instanceof Ninja == true){
```

Comment:
Since `opponent instanceof Ninja` returns a boolean we don't need to compare it to `true`.